modules/nw-sriov-interface-level-sysctl-basic-node-policy.adoc

```diff
@@ -0,0 +1,79 @@
+
+The SR-IOV Network Operator adds the `SriovNetworkNodePolicy.sriovnetwork.openshift.io` custom resource definition (CRD) to {product-title}. You can configure an SR-IOV network device by creating a `SriovNetworkNodePolicy` custom resource (CR).  
+
+[NOTE]
+====
+When applying the configuration specified in a `SriovNetworkNodePolicy` object, the SR-IOV Operator might drain and reboot the nodes.
+
+It can take several minutes for a configuration change to apply.
+====
+
+Follow this procedure to create a `SriovNetworkNodePolicy` custom resource (CR).  
+
+.Procedure
+
+. Create an `SriovNetworkNodePolicy` custom resource (CR). For example, save the following YAML as the file `policyoneflag-sriov-node-network.yaml`: 
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: policyoneflag <1>
+  namespace: openshift-sriov-network-operator <2>
+spec:
+  resourceName: policyoneflag <3>
+  nodeSelector: <4>
+    feature.node.kubernetes.io/network-sriov.capable="true"
+  priority: 10 <5>
+  numVfs: 5 <6>
+  nicSelector: <7>
+    pfNames: ["ens5"] <8>
+  deviceType: "netdevice" <9>
+  isRdma: false <10>
+----
++
+<1> The name for the custom resource object.
+<2> The namespace where the SR-IOV Network Operator is installed.
+<3> The resource name of the SR-IOV network device plug-in. You can create multiple SR-IOV network node policies for a resource name.
+<4> The node selector specifies the nodes to configure. Only SR-IOV network devices on the selected nodes are configured. The SR-IOV Container Network Interface (CNI) plug-in and device plug-in are deployed on selected nodes only.
+<5> Optional: The priority is an integer value between `0` and `99`. A smaller value receives higher priority. For example, a priority of `10` is a higher priority than `99`. The default value is `99`.
+<6> The number of the virtual functions (VFs) to create for the SR-IOV physical network device. For an Intel network interface controller (NIC), the number of VFs cannot be larger than the total VFs supported by the device. For a Mellanox NIC, the number of VFs cannot be larger than `128`.
+<7> The NIC selector identifies the device for the Operator to configure. You do not have to specify values for all the parameters. It is recommended to identify the network device with enough precision to avoid selecting a device unintentionally.
+If you specify `rootDevices`, you must also specify a value for `vendor`, `deviceID`, or `pfNames`. If you specify both `pfNames` and `rootDevices` at the same time, ensure that they refer to the same device. If you specify a value for `netFilter`, then you do not need to specify any other parameter because a network ID is unique.
+<8> Optional: An array of one or more physical function (PF) names for the device.
+<9> Optional: The driver type for the virtual functions. The only allowed values are `netdevice` and `vfio-pci`. The default value is `netdevice`.
```

Comment:
@kquinn1204 @mmirecki please be aware that for the sysctl you can't use vfio-pci it will not work. so maybe you can add a small note here?

Reply:
@SchSeba I created this https://issues.redhat.com/browse/OCPBUGS-3638 to track